lib/livebook/runtime/evaluator.ex

```diff
@@ -454,6 +455,26 @@ defmodule Livebook.Runtime.Evaluator do
   defp code_error?(%CompileError{}), do: true
   defp code_error?(_error), do: false
 
+  defp reorder_binding(binding, prev_binding) do
+    # We keep the order of existing binding entries and move
+    # the new ones to the beginning, ordered alphabetically
```

Comment:
Do we need to keep the alphabetical ordering?

Reply:
We don't, but probably better this way than randomly, no?

Reply:
I'm thinking mostly in terms of Kino UI, if a cell defines both variables `df1` and `df2`, it feels more natural if the subsequent smart cell to defaults to `df1`. And in most cases a cell would define a most a couple bindings, so sorting shouldn't be a burden.